ref(profiling): create useDifferentialFlamegraphQuery

JonasBa opened the pull request:
Create a data loading hook for differential flamegraph data loading

JonasBa committed:
feat(profiling): add differential flamegraph boilerplate to statistical issue

JonasBa committed:
ref(profiling): modify aggregate flamegraph hook to accept selection params (#59909)

Decouple useAggregateFlamegraphQuery from implicit filter selection

JonasBa committed:
Merge branch 'master' into jb/profiling/differential-flamegraph

JonasBa committed:
Merge branch 'master' into jb/profiling/differential-flamegraph

JonasBa committed:
ref(profiling): create useDifferentialFlamegraphQuery

JonasBa committed:
Merge branch 'master' into jb/profiling/differential-flamegraph-data

JonasBa committed:
Merge branch 'master' into jb/profiling/differential-flamegraph-data

Zylphrex commented on static/app/components/events/eventStatisticalDetector/eventDifferentialFlamegraph.tsx:
```diff
@@ -32,6 +32,16 @@ export function EventDifferenialFlamegraph(props: EventDifferenialFlamegraphProp
+
+  // @todo use the data to render a flamegraph
+  useDifferentialFlamegraphQuery({
+    projectID: isNaN(projectID) ? -1 : projectID,
```
I don't think -1 is accepted here. You should disable the query instead

JonasBa committed:
fix(profiling): disable differential query

JonasBa committed:
ref(profiling): create transaction delta provider (#59998)

Affected txns data will be reused between the affected txn table and
differential flamegraph so hoist the data fetching outside of affected
txns table component

JonasBa committed:
Merge branch 'master' into jb/profiling/differential-flamegraph-data

JonasBa committed:
Merge branch 'master' into jb/profiling/differential-flamegraph-data

JonasBa committed:
Merge branch 'master' into jb/profiling/differential-flamegraph-data
